ChangeLog

```diff
@@ -1,3 +1,6 @@
+Version 5.2.1
```

Comment:
I feel snapshot is something that is not there yet and the new package is going to remove snapshot.